test_unstructured/partition/test_text.py

```diff
@@ -514,3 +514,37 @@ def test_add_chunking_strategy_on_partition_text(filename="example-docs/norwich-
     chunks = chunk_by_title(elements)
     assert chunk_elements != elements
     assert chunk_elements == chunks
+
+
+def test_partition_text_element_metadata_has_languages():
+    filename = "example-docs/norwich-city.txt"
+    elements = partition_text(filename=filename)
+    assert elements[0].metadata.languages == ["eng"]
+
+
+# def test_partition_text_detects_quote_in_other_language():
```

Comment:
@shreyanid 
`langdetect` doesn't seem to be sensitive enough to detect quotes in other languages, so let me know if these tests can be removed.

Reply:
Perhaps we should leave them in with an xfail (or similar result) so we remember that this is functionality that is not yet supported, and can revisit other solutions in the future.